src/app/dashboard_view/dashboard.component.spec.ts

```diff
@@ -148,6 +153,11 @@ describe('Dashboard', () => {
         By.directive(ChartComponent))[1].componentInstance;
     });
 
+    afterEach(()=>{
```

Comment:
is there any specific reason that you need the same afterEach twice?

Reply:
It's placed in 2 separate describe blocks. One for Bar Chart and one for Pie Chart